base/domain-xho.yml

```diff
@@ -0,0 +1,377 @@
+
+          YAMKELA ✅ - _ukwamkela imigaqo nemiqathango_
+          OKUNINZI 📄 - -ukufunda imigaqo nemiqathango_
+
+          (okanye phendula iMENYU 📌 _ukubuyela kwimenyu yakuqala_)
+
+  utter_more_terms:
+      - text: |
+          Uyaqinisekisa ukuba unoxanduva kunyango lwakho nokhathalelo. Le nkonzo ibonelela ngolwazi kuphela.
+
+          Le nkonzo ayithathi indawo yengcebiso kwezonyango / ukufumanisa isifo / unyango. Fumana ingcebiso kumboneleli wezempilo ofanelekileyo malunga nemeko yonyango / ukhathalelo lwakho, ingakumbi ukuba uneempawu ezixhalabisayo kakhulu.
+
+          Uyaqinisekisa ukuba awuzukungahoyi / ulibazise ukufuna ingcebiso zempilo malunga nonyango/ nokhathalelo ngenxa yale nkonzo.Thembela kolu lwazi ngokusemngciphekweni wakho.
+
+          Phendula ngegama elinye kula *ANGONOBUMBA* ukuze uqhubekeke
+          *YAMKELA* ✅ ukukwamkela imigaqo nemiqathango
+          (okanye phendula u 📌 *IMENYU* ukubuyela kwimenyu yakuqala)
+
+  utter_ask_province:
+      - text: |
+          Uhlala kweliphi iphondo ngoku?
+
+          Phendula:
+          *1.* eMPUMA KOLONI
+          *2.* eFREYISTATA
+          *3.* eGAUTENG
+          *4.* KWAZULU NATAL
+          *5.* eLIMPOPO
+          *6.* eMPUMALANGA
+          *7.* eMNTLA-NTSHONA
+          *8.* eMNTLA KOLONI
+          *9.* eNTSHONA KOLONI
+
+  utter_incorrect_selection:
+    - text: |
+        Le nkonzo isebenza kakuhle xa usebenzisa ukhetho olunikezwe ngo *NOBUMBA*.
+        📌Phendula u *IMENYU* ukubuyela kwimenyu yakuqala
+
+  utter_incorrect_location:
+    - text: |
+        If you have typed your address incorrectly, please try again. If you are unable to provide your address, please TYPE the name of your Suburb, Township, Town or Village (or nearest)
+
+  utter_ask_age:
+    - text: |
+        Uneminyaka emingaphi?
+
+        *1.* <18
+        *2.* 18-39
+        *3.* 40-65
+        *4.* >65
+
+  utter_ask_gender:
+    - text: |
+        Nceda usinike isini ozikhomba ngaso:
+
+        Phendula
+        *1.* INDODA
+        *2.* IBHINQA
+        *3.* OKUNYE
+        *4.* KHETHA UKUNGATSHO
+
+  utter_ask_location:
+    - text: |
+        Sidinga ukuqokelela indawo yakho.
+
+        Nceda yabelana ngendawo yakho 📍 usebenzisa u-WhatsApp ngokulandela lamanyathelo:
+        - Cofa i-ayikhoni yokuncamathisela (+ okanye 📎) ezantsi ku-WhatsApp
+        - Khetha indawo
+        - Oku kuya kuzisa imephu enepini ebonisa indawo okuyo ngoku. Cofa *Thumela Indawo yakho okuyo ngoku* ukuba le yidilesi yakho
+        - Unokukhetha nakwezinye iindawo ezikufuphi ukuba ucinga ukuba iGPS iphume ecaleni kancinci.
+
+        Okanye TAYIPA igama lommandla wakho, iLokishi, iDolophu okanye iLali (okanye ekufutshane)?
+
+  utter_ask_location_confirm:
+    - text: |
+        ⬛⬛⬛⬜⬜⬜⬜⬜⬜⬜⬜
+
+        Sisebenzisa Iinkonzo zeNdawo zikaGoogle ukuqinisekisa iindawo. Nceda uqinisekise ukuba idilesi engezantsi ichanekile ngokusekelwe kulwazi olwabeleyo:
+        {location}
```

Comment:
This location needs to be surrounded by the `----------------------------` characters